src/StackHCI/help/Set-AzStackHCI.md

```diff
@@ -0,0 +1,281 @@
+---
+external help file: Az.StackHCI-help.xml
+Module Name: Az.StackHCI
+online version: https://docs.microsoft.com/powershell/module/az.stackhci/set-azstackhci
+schema: 2.0.0
+---
+
+# Set-AzStackHCI
+
+## SYNOPSIS
+Set-AzStackHCI modifies resource properties of the Microsoft.AzureStackHCI cloud resource representing the on-premises cluster to enable or disable features.
+
+## SYNTAX
+
+```
+Set-AzStackHCI [[-ComputerName] <String>] [-Credential <PSCredential>] [-ResourceId <String>]
+ [-EnableWSSubscription <Boolean>] [-DiagnosticLevel <DiagnosticLevel>] [-TenantId <String>]
+ [-ArmAccessToken <String>] [-GraphAccessToken <String>] [-AccountId <String>] [-EnvironmentName <String>]
+ [-UseDeviceAuthentication] [-Force] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Set-AzStackHCI modifies resource properties of the Microsoft.AzureStackHCI cloud resource representing the on-premises cluster to enable or disable features.
+
+## EXAMPLES
+
+### EXAMPLE 1
+```powershell
+PS C:\\\> Set-AzStackHCI -EnableWSSubscription $true
+```
+
+```output
+Result: Success
+```
```

Comment:
@mikefrobbins Hey Mike, I haven't seen this format before, will docs.microsoft.com render it correctly?

Reply:
I can just set it to be the same as register-* already is as part of the other update below

Reply:
Though, many other cmdlet help files are already using it as well, for instance:
https://raw.githubusercontent.com/Azure/azure-powershell/main/src/Accounts/Accounts/help/Connect-AzAccount.md

Reply:
Alright, considering `Connect-AzAccount` is rendered correctly on docs.microsoft.com I suppse it's OK